src/actions/tests/pause.spec.js

```diff
@@ -23,27 +33,68 @@ const mockThreadClient = {
 };
 
 describe("pause", () => {
-  it("should set and clear the command", async () => {
-    const { dispatch, getState } = createStore(mockThreadClient);
-    const mockPauseInfo = {
-      frames: [{ id: 1, scope: [], location: { sourceId: "foo1", line: 4 } }],
-      loadedObjects: [],
-      why: {}
-    };
-
-    await dispatch(actions.paused(mockPauseInfo));
-    dispatch(actions.stepIn());
-    expect(isStepping(getState())).toBeTruthy();
-    await stepInResolve();
-    expect(isStepping(getState())).toBeFalsy();
+  describe("paused", () => {
+    it("should detect when we're paused due to an expression exception", async () => {
+      const { dispatch, getState } = createStore(mockThreadClient);
+      const mockPauseInfo = {
+        frames: [{ id: 1, scope: [], location: { sourceId: "foo1", line: 4 } }],
+        loadedObjects: [],
+        why: { type: "exception" }
+      };
+
+      dispatch(actions.addExpression("foo"));
```

Comment:
could you turn "foo" into "foo.bar" so we understand it is supposed to throw ? 
Also, do we need this dispatch and the other one ? It seem like it's `actions.paused` that really has value here.